academy/ibc-dev/custom-ibc-application.md

```diff
@@ -1,3 +1,5 @@
+<!-- DO NOT INCLUDE -->
```

Comment:
We can not put anything above the header, otherwise the header will not be loaded correctly. I'll move this below the header.